index.js

```diff
@@ -124,30 +122,31 @@ module.exports = function (spec) {
       return pad(format.red(symbols.cross + ' No tests found'));
     }
 
-    return _.filter([
+    const temp = [
       pad('total:     ' + results.asserts.length),
       pad(format.green('passing:   ' + results.pass.length)),
       results.fail.length > 0 ? pad(format.red('failing:   ' + results.fail.length)) : undefined,
       pad('duration:  ' + prettyMs(new Date().getTime() - startTime))
-    ], _.identity).join('\n');
+    ];
+    return temp.filter( value => typeof value !== 'undefined' ).join('\n');
   }
 
   function formatFailedAssertions (results) {
 
     var out = '';
 
-    var groupedAssertions = _.groupBy(results.fail, function (assertion) {
+    var groupedAssertions = results.fail.filter(function (assertion) {
       return assertion.test;
     });
 
-    _.each(groupedAssertions, function (assertions, testNumber) {
+    groupedAssertions.forEach(function (assertions, testNumber) {
 
       // Wrie failed assertion's test name
-      var test = _.find(results.tests, {number: parseInt(testNumber)});
+      var test = results.tests.find(results.tests, {number: parseInt(testNumber)});
```

Comment:
Should be something like this:
```js
results.tests.find(function(v){ return v.number == parseInt(testNumber); });
```

Reply:
fixed, thanks for spotting this, I was going too quickly.